algebird-core/src/main/scala/com/twitter/algebird/QTree.scala

```diff
@@ -200,6 +200,20 @@ case class QTree[A](
       (0L, 0L)
     }
   }
+  /**
+   * For a given value and given quantiles list, compute which quantile it fails to in this QTree
+   * @param value The specified value to compute quantile.
+   * @param quantiles The specified quantiles list, need to be sorted.
+   */
+  def getValueQuantile(value: Double, quantiles: Seq[Double]): (Double, Double) = {
+    val quantilePoints = quantiles.map { q: Double =>
+      val (lower, upper) = quantileBounds(q)
+      (lower, upper, q)
+    }
```

Comment:
I'm thinking situations like this:
Given an ordered and unique list of quantiles, e.g. [0, 0.1, 0.2, 0.3,... 0.9, 1.0] and specific value x. This method will return the estimated quantile for x in this QT.  Wondering if this is a valid and common use case?

Since the list is ordered and unique, there will be only one item in the list.  Also It's possible that x's quantile is somewhere b/w the two items in the list. For example, x's quantile is 0.25 which is in the middle of [0.2, 0.3]. In this case, I think it should return the the interval as lower and upper bound. So that's why the algorithm uses `>=` to do the filtering. The `1.0` is because if no bounds in the QT is larger than value x, then I think its quantile should be 1.0 since it's larger than all values in this QT.
Does this make sense?